lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -49,6 +49,36 @@ var DataObservatoryMultipleMeasures = Node.create(TYPE, PARAMS, { cache: true, l
             throw new Error('The number of numerators=' + this.numerators.length +
                 ' does not match the number of numerator_timespans=' + this.numerator_timespans.length);
         }
+
+        this.columnTypes = this.numerators.reduce((columnTypes, numeratorId, index) => {
```

Comment:
I want to default all columns to `::Numeric`, the current behaviour, and later in the async code, only change the columns that I found based on the `numerator_id`, that way I don't have to expect responses in the same order, or metadata for all measures. I want to play safe here.